code/datums/mutations/actions.dm

```diff
@@ -502,3 +502,43 @@
 	spikey.forceMove(get_turf(L))
 	transfered.visible_message("<span class='notice'>[spikey] falls out of [transfered]!</span>")
 
+/datum/mutation/human/nv
+	name = "Night Vision"
+	desc = "The user of this genome can see well in the dark."
+	quality = POSITIVE
+	text_gain_indication = "<span class='notice'>Your eyes feel different...</span>"
+	instability = 40
+	locked = TRUE
+	power = /obj/effect/proc_holder/spell/self/nvtoggle
+
+/obj/effect/proc_holder/spell/self/nvtoggle
+	name = "Night Vision"
+	desc = "Focus really hard to adjust your eyesight."
+	clothes_req = NONE
+	antimagic_allowed = TRUE
+	charge_max = 100
+	active = FALSE
+	action_icon = 'icons/mob/actions/actions_changeling.dmi'
+	action_icon_state = "ling_augmented_eyesight"
+
+/obj/effect/proc_holder/spell/self/nvtoggle/cast(mob/living/carbon/human/user)
+	. = ..()
+	var/obj/item/organ/eyes/E = user.getorganslot(ORGAN_SLOT_EYES)
+	if(E)
+		if(!active)
+			ADD_TRAIT(user, TRAIT_TRUE_NIGHT_VISION, GENETIC_MUTATION)
```

Comment:
Doesn't this mean you can activate this, then remove your eyes and replace them with normal ones to gain night vision without any negatives? Should probably adjust the eyes' see_in_dark instead, plus react to them being removed.

Reply:
This was already very confusing to do when I did it 6 months ago and I have no idea how to adjust all that much about it anymore.

Reply:
I can take a look soonish:tm: and see what I can do, ping me tomorrow to remind me if I'm not done by then.

Reply:
Any word on this? Changing see_in_dark would be very easy